ldflags broken?

82d548aa540b3fa25add9fca24e81cb2cdbc63cf apparently broken ldflags entirely; even after c9219b8cd01bacd440e9cab950ea32d6d7cebf23 I get:

```$ astilectron-bundler -v -l -ldflags "X:go.zr.org/auth/zam/internal/version.Version=xyzzy" -d -o "/home/frew/go/src/go.zr.org/auth/zam/output"
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/asticode/go-astilectron-bundler.LDFlags.Merge(0x0, 0xc000083560)
        /home/frew/go/src/github.com/asticode/go-astilectron-bundler/ldflags.go:36 +0x12a
main.main()
        /home/frew/go/src/github.com/asticode/go-astilectron-bundler/astilectron-bundler/main.go:90 +0x387
```

I'm going to configure our build system to just use our fork for the moment; I don't really understand the motivation of the first change, but a quick fix is:

```golang
diff --git a/astilectron-bundler/main.go b/astilectron-bundler/main.go
index 3136b2e..5458477 100644
--- a/astilectron-bundler/main.go
+++ b/astilectron-bundler/main.go
@@ -87,6 +87,9 @@ func main() {
        }
 
        // Flags
+       if c.LDFlags == nil {
+               c.LDFlags = astibundler.LDFlags(make(map[string][]string))
+       }
        c.LDFlags.Merge(astibundler.LDFlags(ldflags))
 
        // Build bundler
```

If you're happy with the above I can submit a PR.

I am curious though, what is the benefit of merging this way?

--- a/astilectron-bundler/main.go
+++ b/astilectron-bundler/main.go
@@ -87,6 +87,9 @@ func main() {
 	}
 
 	// Flags
+	if c.LDFlags == nil {
+		c.LDFlags = astibundler.LDFlags(make(map[string][]string))
+	}
 	c.LDFlags.Merge(astibundler.LDFlags(ldflags))
 
 	// Build bundler

